templates/persistent-volume.yaml

```diff
@@ -18,7 +18,6 @@ spec:
   hostPath:
     # Directory location on host
     path: {{ required "need to define the policy directory by using --set global.persistence.hostpath=localpath location" .Values.global.persistence.hostpath  | quote }}
-    type: Directory
```

Comment:
Why has this changed?

Reply:
Latest release of docker desktop has deprecated this out so it needs to go if you are to deploy from now on. Should not have gone in in the first place.